app/controllers/work_decommission_controller.rb

```diff
@@ -27,6 +28,10 @@ def decommission(work)
     work.head.decommission!
   end
 
+  def send_decommission_email(work)
+    WorksMailer.with(work: work).decommission_email.deliver_later
```

Comment:
Can this be added as a hook on the workflow transition rather than in the controller? That is how most of our emails are triggered.

Reply:
I've updated this as suggested.